apps/admin/src/foster-families/db.server.ts

```diff
@@ -100,6 +108,31 @@ export class FosterFamilyDbDelegate {
     });
   }
 
+  async setIsBanned(fosterFamilyId: FosterFamily["id"], isBanned: boolean) {
+    await prisma.$transaction(async (prisma) => {
+      try {
+        await prisma.fosterFamily.update({
+          where: { id: fosterFamilyId },
+          data: {
+            isBanned,
+            availability: isBanned ? "UNAVAILABLE" : "UNKNOWN",
```

Comment:
Prisma génère des enum pour nous :

```suggestion
            availability: isBanned
              ? FosterFamilyAvailability.UNAVAILABLE
              : FosterFamilyAvailability.UNKNOWN,
```